OfficeUIFabricCore/OfficeUIFabricCore/Customization/UITextField/UIImageMSTextFieldExtension.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+import UIKit
+
+extension UIImage {
+    class func msTextFieldBoxBackground(borderColor: UIColor, fillColor: UIColor, scale: CGFloat = UIScreen.mainScreen().scale) -> UIImage {
+        
+        let innerImageInsets: CGFloat = 1
```

Comment:
@chedon they are local constants, and we should be able to change them separately later